cluster/addons/rbac/cluster-loadbalancing/glbc/clusterrole.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:controller:glbc
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+rules:
+- apiGroups: [""]
+  resources: ["secrets", "endpoints", "services", "pods", "nodes", "namespaces", "configmaps", "events"]
+  verbs: ["get", "list", "watch", "update", "create", "patch"]
+- apiGroups: ["extensions"]
+  resources: ["ingresses"]
+  verbs: ["get", "list", "watch", "update"]
```

Comment:
this also surprises me

Reply:
looks like it updates annotations: https://github.com/kubernetes/ingress-gce/blob/50d49b077d9ab4362a02fae05f94e433cd3f08dc/pkg/controller/controller.go#L579

Reply:
it can (and should) update annotations via the ingress/status subresource. separate out the update permission into its own rule, open a follow-up item to switch to using the status subresource for that, and reference the issue in a comment in the rule